isso/js/app/isso.js

```diff
@@ -296,16 +270,20 @@ var insert = function(comment, scrollIntoView) {
 
             toggler.canceled = false;
             api.view(comment.id, 1).then(function(rv) {
-                var textarea = editorify($.new("div.isso-textarea"));
+                var textarea = $.new("textarea.isso-textarea");
+                textarea.setAttribute("rows", 10);
+                textarea.setAttribute("maxlength", 65535);
 
-                textarea.innerHTML = utils.detext(rv.text);
+                textarea.value = rv.text;
                 textarea.focus();
 
                 text.classList.remove("isso-text");
                 text.classList.add("isso-textarea-wrapper");
 
                 text.textContent = "";
                 text.append(textarea);
+
+                textarea.scrollIntoView(); // manually scroll the full textbox into view on initial resize
```

Comment:
this is not "on initial resize" but rather when returning from preview to editing again, no?

Reply:
This function triggers once when you press the Edit link on an existing comment. I originally added this line when I was using `autosize` to resize the textbox, and when it did that, then the first resize of the textarea would make it scroll off screen if the comment was longer than a few lines.

It actually looks like this serves a different function now. When editing without autosize, it still is halfway off-screen when editing I'll update the comment, let me know what you think about this addition.